tests/integration/defs/triton_server/test_triton_llm.py

```
@@ -3735,6 +3735,88 @@ def test_llmapi_backend(E2E_MODEL_NAME, DECOUPLED_MODE, TRITON_MAX_BATCH_SIZE,
+    print_info(f"DEBUG:: launch_server with args: {cmd}")
+    check_call(cmd, shell=True)
+    check_server_ready()
+
+    # Test with grpc protocol and streaming (decoupled mode)
+    protocol = "grpc"
+
+    # Run end_to_end_test
+    run_cmd = [
+        f"{llm_backend_inflight_batcher_llm_root}/end_to_end_test.py",
+        f"--protocol={protocol}",
+        f"--test-llmapi",
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)